macos-example/macos/Podfile.lock

```diff
@@ -1053,7 +1053,7 @@ PODS:
     - React-jsi (= 0.73.32)
     - React-logger (= 0.73.32)
     - React-perflogger (= 0.73.32)
-  - RNReanimated (3.9.0):
+  - RNReanimated (3.9.0-nightly-20240421-474f6983d):
```

Comment:
Should it end up here?

Reply:
Somehow, the `main` branch had wrong `Podfile.lock` and the CI were failing for this one without the update.